src/main/java/org/broadinstitute/hellbender/tools/spark/sv/StructuralVariationDiscoveryArgumentCollection.java

```diff
@@ -104,6 +104,14 @@
         @Argument(doc = "Adapter sequence.", fullName = "adapter-sequence", optional = true)
         public String adapterSequence;
 
+        @Argument(doc = "Minimum classified probability for a piece of evidence to pass xgboost evidence filter",
+                fullName = "sv-evidence-filter-threshold-probability")
+        public double svEvidenceFilterThresholdProbability = 0.3;
```

Comment:
The number was this low because the data set was imbalanced (it favored correct classification of "bad" evidence rather than "good", and therefore was a poor estimator on the data we actually care about). I updated training parameters to correct for that, and the corresponding updated threshold is much closer to 1.0. It should therefore intuitively correspond to high confidence that any given piece of evidence is good.